internal/client/console/console_client.go

```diff
@@ -0,0 +1,19 @@
+package console
+
+import (
+	"bufio"
+	"haracho-go/internal/client"
+	_ "haracho-go/internal/command"
+	"strings"
+)
+
+type Client struct {
+	Scanner *bufio.Scanner
+}
+
+func (c Client) Parse() (*client.Arg, client.CommandContext) {
+	c.Scanner.Scan()
+	input := c.Scanner.Text()
+	split := strings.Split(input, " ")
+	return &client.Arg{CommandName: split[0], Args: split[1:]}, CommandContext{}
```

Comment:
```suggestion
	return client.NewArg(input), CommandContext{}
```
別記の通りに `Arg` を直すとこうなる